core/View.php

```diff
@@ -93,11 +95,7 @@ public function setView($requestedView = false, $renderName = false)
                 if (!file_exists($view)) {
                     throw new \Exception('View: "' . $view . '" could not be found.');
                 }
-                if($renderName !== false) {
-                    $this->$renderName = $this->process($view);
-                } else {
-                    $this->view = $this->process($view);
-                }
+                $this->view = $view;
             } else {
                 throw new \Exception('Please provide a view for setView.');
```

Comment:
Sometimes I just render the layout (login for example), but yeah, a view will always be called inside of here.